package.json

```diff
@@ -94,10 +94,11 @@
   "scripts": {
     "benchmark-rule": "node scripts/benchmark-rule.js",
     "flow": "flow",
-    "jest": "jest",
+    "jest": "jest --coverage",
     "lint:js": "eslint . --cache",
     "lint:md": "remark . --quiet --frail",
     "lint": "npm-run-all --parallel lint:*",
+    "preci": "npm-run-all --serial lint:md flow",
```

Comment:
Why is this required? Why not use the existing `pretest` considering you're pulling in `eslint-config-stylelint`:

> _"I've configured the prepare block in the Code Climate configuration file to take advantage of this feature and pull down your shared ESLint configuration."_